src/Intersection.jl

```diff
@@ -439,6 +438,34 @@ function ρ(d::AbstractVector{N}, cap::Intersection{N, S1, S2}; kwargs...
                    for Hi in constraints_list(cap.Y)])
 end
 
+"""
+    ρ(d::AbstractVector{N}, cap::Intersection{N, S1, S2}; kwargs...
+     ) where {N<:Real, S1<:AbstractPolyhedron{N}, S2<:AbstractPolyhedron{N}}
+
+Return an upper bound on the support function of the intersection between two
+polyhedral sets.
+
+### Input
+
+- `d`      -- direction
+- `cap`    -- intersection of two polyhedral sets
+- `kwargs` -- additional arguments that are passed to the support-function
+              algorithm
+
+### Output
+
+The support function for the given direction.
+
+### Algorithm
+
+We combine the constraints of the two polyhedra to a new `HPolyhedron`, for
+which we then evaluate the support function.
+"""
+function ρ(d::AbstractVector{N}, cap::Intersection{N, S1, S2}; kwargs...
```

Comment:
i dont remember if `kwargs` here is intentional or can be removed

Reply:
Other methods have that, so we should keep it for consistency of the interface (although that is already quite messed up).